_data/twenty_20/faqs.yml

```diff
@@ -0,0 +1,22 @@
+- question: "How much does participation cost?"
+  answer: "There is no participation fee. Lunches and coffee breaks will be
+  covered."
+
+- question: "Is there anything else for free?"
+  answer: "We can offer a limited number of rooms during the school (four
+  nights). For more details, see the <a href=\"../register\">registration
+  page</a> (<em>the registration deadline was August 31</em>)."
+
+- question: "How much do I need to know about model reduction?"
+  answer: "Nothing. Basic knowledge of linear algebra and differential equations
+  is expected."
```

Comment:
Did we consider increasing the requirements? Or just doing shorter lectures?

Reply:
Just shorter lectures I think

Reply:
Ok, that's probably better.